PWGHF/D2H/TableProducer/candidateCreatorCharmResoReduced.cxx

```diff
@@ -172,6 +175,8 @@ struct HfCandidateCreatorCharmResoReduced {
     massDplus = o2::constants::physics::MassDPlus;
     massDstar = o2::constants::physics::MassDStar;
     massD0 = o2::constants::physics::MassD0;
+    massPion = o2::constants::physics::MassPiPlus;
+    massKaon = o2::constants::physics::MassKPlus;
```

Comment:
This is useless. Just add `using namespace o2::constants::physics;` and use the constants directly.